packages/ui/src/kit/VTabs/VTabs.vue

```diff
@@ -1,27 +1,37 @@
 <template>
-  <div :class="vTabsCssClass">
+  <div
+    v-if="$slots.default"
+    :class="vTabsCssClass"
+  >
     <section class="v-tabs-controls-list">
       <button
-        v-for="(child, index) in $slots.default.filter($el => !!$el.tag)"
+        v-for="(tab, index) in $slots.default.filter($el => !!$el.tag)"
```

Comment:
> $slots.default.filter($el => !!$el.tag)

Maybe computed? Something like `childTabs`. Will be more declarative.

Reply:
I will be not reactive.